specification/app/data-plane/Microsoft.App.DynamicSessions/preview/2024-10-02-preview/examples/CodeExecution_ExecuteCode_Async.json

```diff
@@ -0,0 +1,32 @@
+{
+  "title": "CodeExecution_ExecuteCode_Async",
+  "operationId": "CodeExecution_ExecuteCode",
+  "headers": {
+    "execution-id": "testexecutionid"
+  },
+  "parameters": {
+    "api-version": "2024-10-02-preview",
+    "sessionId": "testidentidier",
+    "codeExecutionRequest": {
+      "codeInputType": "Inline",
+      "executionType": "Asynchronous",
+      "code": "print(7*9)",
+      "timeoutInSeconds": 60
+    }
+  },
+  "responses": {
+    "202": {
+      "headers": {
+        "Operation-Location": "location"
+      },
+      "body": {
```

Comment:
This example seems outdated. please update it with latest contracts

Reply:
Updated